android/src/main/java/com/appcuesreactnative/ReactNativeElementTargeting.kt

```diff
@@ -52,9 +51,10 @@ private fun View.asCaptureView(): ViewElement? {
     }
 
     // ignore the Appcues SDK content that has been injected into the view hierarchy
-    if (this.isAppcuesView()) {
-        return null
-    }
+    // NOTE: revisit this when Android SDK is updated to expose this function publicly
+    // if (this.isAppcuesView()) {
+    //     return null
+    // }
```

Comment:
this was accidentally made internal, rather than public, in a recent android native SDK update that went into the 2.0.0 release

we can revisit this in a later update for element targeting, but for now, commenting out to ensure a non-broken react native module for android builds